.huskyrc

```diff
@@ -1,5 +1,5 @@
 {
 	"hooks": {
-		"pre-commit": "grunt configure && lint-staged"
```

Comment:
This is temporary as I converted the code, will be reverted once eslint and grunt are setup properly